jerry-core/ecma/base/ecma-helpers-conversion.c

```diff
@@ -818,12 +818,10 @@ ecma_uint32_to_utf8_string (uint32_t value, /**< value to convert */
  *
  * @return number - result of conversion.
  */
-ecma_number_t
+ecma_number_t __attr_always_inline___
```

Comment:
I wouldn't add the attributes then. IMHO, the cleaner the code the better (i.e., we should avoid using attributes as much as we can).